app/gateway/2.7.x/configure/auth/kong-manager/oidc-mapping.md

```diff
@@ -37,9 +37,13 @@ to identify which admin value and role name to map from the IdP to {{site.base_g
 The `admin_claim` value specifies which IdP username value should map to Kong Manager. 
 The username and password are required for the user to log into the IdP.
 
-The `authenticated_groups_claim` value specifies which IdP claim should be assigned to the
-specified {{site.base_gateway}} user. The exact value depends on your IdP -- for example, Okta 
-configures claims for `groups`, and other IdPs might configure them as `roles`.
+The `authenticated_groups_claim` value specifies which IdP claim should be used to assign {{site.base_gateway}} roles to the
+specified {{site.base_gateway}} admin. 
+
+- The group claim name depends on your IdP -- for example, Okta configures claims for `groups`, and other IdPs might configure them as `roles`. 
+- The group claim value need to follow the format `<workspace_name>:<role_name>` for dynamic role assigning. 
```

Comment:
```suggestion
This value depends on your IdP -- for example, Okta configures claims for `groups`, and another IdP might configure them as `roles`.

In the Idp, the claim must follow the format `<workspace_name>:<role_name>`.
```

Reply:
When we are talking about payload of a JWT token, we refer a `key:value` pair as a claim. 

So to make it clearer, I chose the wording as `<claim_name>:<claim_value>`.

What customer need to do is to make sure they map `authenticated_groups_claim` to the right `claim_name` and the `claim_value` must follow the format of `<workspace_name>:<role_name>`.

IMO it is not clear enough to say the ``claim must follow the format of `<workspace_name>:<role_name>` ``, it is the `claim_value` need to follow this format. 

Also both claim and value depend on what users have on IdP (not just the value).

For example, in keycloak I can use **kong-group** as the claim name for groups membership in my application.